lib/ui/playground/data/playground_tasks_datasource.dart

```diff
@@ -101,16 +107,28 @@ class PlayGroundTasksDataSource implements PlaygroundDataSource {
           ),
         ),
       ),
-      /* Template
       PlaygroundWidget(
-        title: '#TASK NAME#',
-        child: YourWidget(),
-      ),*/
-      PlaygroundWidget(title: "FARM-280 Update Discover", child:
-          ArticleList(
-              viewModelProvider: ArticleListProvider( title: "Test",
-                  repository: MockArticlesRepository(articleCount: 2000)))
-      )
+        title: 'TASK FARM-59 Record a Sale',
+        child: RecordAmount(viewModel: MockRecordAmountViewModel.buildSale()),
+      ),
+      PlaygroundWidget(
```

Comment:
Lets try and keep one entry per PR, so I would nest these.

Reply:
Done